config/initializers/omniauth.rb

```diff
@@ -17,7 +30,7 @@ def callback_phase
           client.redirect_uri = redirect_uri
           client.authorization_code = authorization_code
           access_token
-          super
+
```

Comment:
@floydprice was this removal intentional? I get a exception on the callback with this removed. An empty array.